src/Akka.Persistence.SqlServer.Tests/Query/SqlServerAllPersistenceIdsSpec.cs

```diff
@@ -0,0 +1,44 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="SqlServerAllPersistenceIdsSpec.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2016 Typesafe Inc. <http://www.typesafe.com>
+//     Copyright (C) 2013-2016 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using Akka.Configuration;
+using Akka.Persistence.Query.Sql;
+using Akka.Persistence.Sql.TestKit;
+using Akka.Util.Internal;
+using Xunit.Abstractions;
+
+namespace Akka.Persistence.SqlServer.Tests.Query
+{
+    public class SqlServerAllPersistenceIdsSpec : AllPersistenceIdsSpec
```

Comment:
We should add [Collection("SqlServerSpec")] attribute to disable parallelization